docs/src/SUMMARY.md

```diff
@@ -17,6 +17,7 @@
     - [Platform Bus for Rust VMM](architecture/adr/platform-bus-passthrough-support.md)
   - [Hardening](architecture/hardening.md)
   - [Secure Boot](architecture/secureboot.md)
+  - [Yubico Passwordless Authentication](architecture/YubicoAuthentication.md)
```

Comment:
Please rename to `(architecture/passwordless_authentication.md)`

Reply:
Updated as per review comment.